packages/approval-controller/src/ApprovalController.ts

```diff
@@ -699,6 +713,17 @@ export class ApprovalController extends BaseControllerV2<
     });
   }
 
+  /**
+   * Sets the loading text for the approval flow.
+   *
+   * @param loadingText - The approval flow loading text that will be displayed.
+   */
+  setFlowLoadingText(loadingText: string | null) {
```

Comment:
I'm equally ok with either this or having `loadingText` as an optional parameter which sets it to null.

Reply:
I vote this, feels like overkill to have options here, at least for now, plus more explicit to provide null instead of an empty object.